crates/shared/src/sources/balancer_v2/pool_fetching.rs

```diff
@@ -163,6 +165,8 @@ pub struct BalancerPoolFetcher {
 #[clap(rename_all = "verbatim")]
 pub enum BalancerFactoryKind {
     Weighted,
```

Comment:
Should we also rename `Weighted -> WeightedV1` and `Stable -> StableV1` now?

Reply:
I kept the names as that is what they are called in the Balancer V2 repo (in order to avoid confusion). I'm also fine with changing it though - no string feelings.

Reply:
I guess it also makes sense to leave it as it is. 👌